subsys/canbus/isotp/Kconfig

```diff
@@ -0,0 +1,136 @@
+# Kconfig - ISO TP configuration options
+
+#
+# Copyright (c) 2019 Alexander Wachter
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig ISOTP
+	bool "ISO-TP Transport for CAN"
```

Comment:
If you think the "CAN" string is too much here, we can certainly shorten that to "ISO-TP Transport"

Reply:
Almost forgot, please add "[EXPERIMENTAL]" at the end of the string so we can more easily change the APIs etc if needed.

Reply:
Good catch!
Need to add it to https://github.com/zephyrproject-rtos/zephyr/projects/15 too when it's merged.